### DISCUSSION TITLE
Ability to copy user prompts

### DISCUSSION THREAD

**Discussion by zavocc** [Taewan-P/gpt_mobile#52] [Ideas]
Right now generated model responses can be copied using "Copy text" button. The ability to copy prompts would be super helpful as it allows me to compare responses from one model to another

**Comment by Taewan-P** [comment]
Do you mean user prompt?

**Comment by zavocc** [comment]
Yeah

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Taewan-P** [comment]
This is implemented by #78.
It will be included in the next release.
Thanks
